### DISCUSSION TITLE
when the wallet app reopen there is a way  reconnect the last session ?

### DISCUSSION THREAD

**Discussion by leeleoo** [WalletConnect/walletconnect-monorepo#979] [Developer Support]
The WalletConnect session disconnection happens once you close your wallet app , but the DApp on your web browser still says your wallet is connected

**Comment by leeleoo** [comment]
well.  I found connector. session

**Reply by tumtum-dev** [reply]
what did you find ? im trying to solve same problem, when i close the app and try to connect again i cant do it, i tried with extendSession and passing all the topics that i saved.

**Reply by glitch-txs** [reply]
@0xQuokka which version of WalletConnect are you using? You should get the session on `provider.session`

**Reply by finessevanes** [reply]
Still blocked?
